Gems/WhiteBox/Code/Source/SubComponentModes/EditorWhiteBoxEdgeRestoreMode.cpp

```diff
@@ -33,6 +34,26 @@ namespace WhiteBox
         // noop
     }
 
+    void EdgeRestoreMode::RegisterActionUpdaters()
+    {
+    }
+
+    void EdgeRestoreMode::RegisterActions()
+    {
+    }
```

Comment:
question: Do we need these if they're not doing anything? Or are they part of an interface we have to include?

Reply:
They are not necessary, but I'm leaving them around for consistency since there's nothing enforcing them at the moment. I'll try to refactor this to make it more streamlined if possible.

Reply:
Okay makes sense for consistency, thanks